app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -341,8 +350,43 @@ export default class CoverageVizBottomSidebar extends React.Component {
     );
   }
 
+  renderNoDataContents() {
+    const { params } = this.props;
+
+    return (
+      <NarrowContainer className={cs.contents}>
+        <div className={cs.header}>
```

Comment:
I have a NoResultsBanner in discovery view, that is similar to this one. I think we should try to consolidate the two of them.

Reply:
Hey, talked to Jenn and we decided it's okay to special case this one for now. It's different from the other one (longer text, different icon), and not clear if it's re-usable like the "NoResultsBanner" is.